SchedulerWebsite/urls.py

```diff
@@ -38,7 +38,10 @@
     path('home/managesection/create', CreateSection.as_view()),
     path('home/managesection/delete', DeleteSection.as_view()),
     path('home/managesection/edit', EditSection.as_view()),
-    path('home/managesection/addta', AddTAToSection.as_view()),
+    path('home/managesection/adduser', AddUserToSection.as_view()),
+    path('home/managesection/adduser/choosesection', ChooseSectionForUser.as_view()),
```

Comment:
See Kevins code how to implement the "choose" parts. He basically just had the same HTML use a "if selected" structure which means that the "choose user" is displayed first, and then when a selection is made, it opens up the next part.